lib/al/Library/Area/AreaShape.cpp

```diff
@@ -0,0 +1,98 @@
+#include "al/Library/Area/AreaShape.h"
+#include "al/Library/Area/AreaShapeCube.h"
+#include "al/Library/Area/AreaShapeCylinder.h"
+#include "al/Library/Area/AreaShapeInfinite.h"
+#include "al/Library/Area/AreaShapeSphere.h"
+#include "al/Library/Math/MathAngleUtil.h"
+#include "al/Library/Math/MathLengthUtil.h"
+#include "al/Library/Matrix/MatrixUtil.h"
+#include <math/seadVector.hpp>
```

Comment:
```suggestion
#include <math/seadVector.h>
```
clang-format to re-order includes